### DISCUSSION TITLE
Erste Anmeldung nach Installation mit Safari übernimmt das vorgeschlagene Passwort nicht

### DISCUSSION THREAD

**Discussion by romeocontrol** [OpenCCU/OpenCCU#1349] [🇩🇪 Fragen&Antworten]
Hallo,
ich hatte das Problem, dass ich mich mit dem nach der Installation eingegebenen Passwort nicht wieder anmelden konnte.
Ich habe dazu das vom Browser und macOS vorgeschlagene Passwort genutzt.
Offensichtlich wird dieses Passwort jedoch nach dem 'sicheres Passwort verwenden-'Dialog nicht übernommen. Per Zufall bin ich darauf gestossen, dass das Passwort einfach leer geblieben ist.
Vorschlag: Passwort manuell eingeben und später in die Passwortverwaltung übernehmen.

**Comment by jp112sdl** [comment]
Kann ich nicht reproduzieren.

Hab es grad mal ausprobiert und nach dem Initialstart der RaspberryMatic das von Safari vorgeschlagene Passwort einsetzen lassen.
Anschließend konnte ich mich als "Admin" mit dem (nun automtisch ausgefüllten) Passwort anmelden.
Das PW wird auch im Schlüsselbund gespeichert.

Getestet mit RM 3.59.6.20210703 (OVA) sowie macOS 11.4 (20F71) und Safari 14.1.1 (16611.2.7.1.4)

**Reply by romeocontrol** [reply]
Danke fürs Testen.
War bei mir mit älterem Mac mit HighSierra, vielleicht lag es daran.
Meist ist das Problem jedoch vor dem Monitor, dessen bin ich mir bewusst ;-)
Heute bekomme ich das Funkmodul, da steht dem nächsten Schritt nichts mehr im Weg.
Grüße
RT
